### DISCUSSION TITLE
How does this project intend to make a profit?

### DISCUSSION THREAD

**Discussion by red-co** [Feel-ix-343/markdown-oxide#180] [General]
- This project looks complex (more than 4 editor plugins), how do you plan to maintain it? How do you plan to make money? I have seen many projects die, even open source projects that received $3 million in venture capital, 
- Software projects and open source projects that have basically only "sponsors" (0 USD income for D language developers), acquisitions by tech companies, funding from venture capital, financial support from large companies,
- In addition, plugins and packages are more difficult to monetize than software. it lacks perceived value, it does not reach the hands of - enthusiastic and willing end consumers, but is provided to the hands of picky developers.
- I feel like open source projects are unsustainable, so when I see projects that interest me I always ask.

**Comment by Feel-ix-343** [comment]
Hey!

Because I use this tool, it is highly likely that I wont stop working on it.

As for monetization, I have had some ideas over time, but none for the near future.

Thank you!
